internal/api/handlers/request_adapters/schedulers.go

```diff
@@ -179,17 +329,111 @@ func FromApiContainerEnvironments(apiEnvironments []*api.ContainerEnvironment) [
 	return environments
 }
 
-func FromApiContainerPorts(apiPorts []*api.ContainerPort) []game_room.ContainerPort {
-	var ports []game_room.ContainerPort
-	for _, apiPort := range apiPorts {
-		port := game_room.ContainerPort{
-			Name:     apiPort.GetName(),
-			Port:     int(apiPort.GetPort()),
-			Protocol: apiPort.GetProtocol(),
-			HostPort: int(apiPort.GetHostPort()),
+func fromApiForwarders(apiForwarders []*api.Forwarder) []*forwarder.Forwarder {
+	var forwarders []*forwarder.Forwarder
+	for _, apiForwarder := range apiForwarders {
+		_forwarder := forwarder.Forwarder{
```

Comment:
Hey @caiordjesus commonly variables with the first character as `_` are created to not be used. What do you think to rename it to something like `entityForwarder`?